pkg/webui/console/components/notifications/index.js

```diff
@@ -21,6 +21,10 @@ import { getNotification } from './utils'
 const NotificationContent = ({ receiver, notificationType, data }) => {
   const NotificationContent = useMemo(() => getNotification(notificationType), [notificationType])
 
+  if (!NotificationContent) {
+    return null
+  }
+
```

Comment:
At first glance it looks like these are not needed and will also leave a ghost notification link in the notification link dropdown with no content. I assume because of the other changes the notification rendering will not reach this point.